packages/dht/src/identifiers.ts

```diff
@@ -16,6 +16,12 @@ export const getRawFromDhtAddress = (address: DhtAddress): DhtAddressRaw => {
     return hexToBinary(address) as unknown as DhtAddressRaw
 }
 
+export const getFlippedDhtAddress = (address: DhtAddress): DhtAddress => {
```

Comment:
```suggestion
export const createDistantDhtAddress = (address: DhtAddress): DhtAddress => {
```

and move to `PeerDiscovery.ts`